tests/phpunit/compat/class-wc-stripe-subscription-renewal-test.php

```
@@ -460,12 +468,21 @@ public function test_renewal_radar_blocked_puts_subscription_on_hold() {
+		$this->assertSame( 0, $mock_subscription->get_date( 'payment_retry' ) );
+
+		// Assert: a Radar note was attached to the subscription.
+		$subscription_notes = $mock_subscription->get_captured_notes();
+		$this->assertNotEmpty( $subscription_notes );
+		$this->assertStringContainsString( 'Stripe Radar blocked this payment', $subscription_notes[0] );
 
 		// Clean up.
 		remove_filter( 'pre_http_request', $pre_http_request_callback );
 		WC_Subscriptions_Helpers::$wcs_get_subscriptions_for_renewal_order = null;
+		WCS_Retry_Manager::reset();
 	}
```

PHPMD (2.15.0)

[error] The method test_renewal_radar_blocked_cancels_pending_retry is not named in camelCase. (undefined)

(CamelCaseMethodName)